src/dbnode/storage/index/postings_list_cache_lru.go

```diff
@@ -121,18 +114,17 @@ func (c *postingsListLRU) Add(
 	var (
 		ent = &entry{
 			uuid:         segmentUUID,
-			pattern:      pattern,
+			query:        query,
```

Comment:
Could the entry have a `key` rather than a `PostingsListCacheQuery` and `patternType`?